src/script/components/MessagesList/Message/DecryptErrorMessage.tsx

```diff
@@ -33,17 +33,18 @@ export interface DecryptErrorMessageProps {
 }
 
 const DecryptErrorMessage: React.FC<DecryptErrorMessageProps> = ({message, onClickResetSession}) => {
-  const {
-    is_resetting_session: isResettingSession,
-    is_recoverable: isRecoverable,
-    user,
-  } = useKoSubscribableChildren(message, ['is_resetting_session', 'is_recoverable', 'user']);
+  const [isResettingSession, setIsResettingSession] = useState(false);
 
   const link = getWebsiteUrl(URL_PATH.DECRYPT_ERROR_1);
-  const caption = t('conversationUnableToDecrypt1', user.name(), {
-    '/highlight': '</span>',
-    highlight: '<span class="label-bold-xs">',
-  });
+  const caption = message.isIdentityChanged
+    ? t('conversationUnableToDecrypt2', message.user().name(), {
+        '/highlight': '</span>',
+        highlight: '<span class="label-bold-xs">',
+      })
+    : t('conversationUnableToDecrypt1', message.user().name(), {
+        '/highlight': '</span>',
```

Comment:
i know this is legacy but is there a reason this needs to have a backslash in the name instead of something like 'closingHighlight'?

Reply:
This is how it is in the string copy. Changing this would discard all the translations we have on crowdin. 

The copy looks like this

```text
[highlight]{{user}}[/highlight]´s device identity changed. Undelivered message.
```